tests/python/frontend/tflite/test_forward.py

```diff
@@ -909,6 +939,24 @@ def representative_data_gen():
     )
 
 
+def test_forward_quantized_convolution():
```

Comment:
Yes, I noticed that it wasn't used e decided to give it a go. I agree it would be a good idea to add general depthwise tests in a follow up. For now I think it is covered by the MobileNet testing, which contains lots of them.

Reply:
https://github.com/apache/tvm/issues/11356